### ISSUE TITLE
display issue when tabbing through Dataset Property Editor window

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#99]
Steps to reproduce:

1) Open the Dataset Property Editor (bug doesn't seem to affect the Dataset Property Viewer window)
2) Move focus to the Country field at the bottom of the left column
3) Hit tab to go to the read-only Status field in the right column
4) The focus on the Status field pushes the rest of the fields out of the window

Tabbing off the Status field brings everything back. Oddly, cycling through the fields backwards with Shift-Tab doesn't have a problem.

You can also see the problem just by clicking on the Status field text (e.g. Imported) if focus isn't in that lower right section already.

**Comment by cseppan** [comment]
Globally setting the preferred size of the Label object is causing problems with warning messages. For instance, when exiting the EMF, the message is cut off as "Do you want to exit the Emission...". When closing an edited dataset without saving, the message in the Discard changes? dialog just shows the word "changes".
